pyproject.toml

```diff
@@ -108,6 +108,7 @@ pycryptodome = "*"
 PyJWT = "*"
 pyserial = "*"
 pyzmq = "*"
+rerun-sdk = "0.16.0"
```

Comment:
ah looks like "0.16.0" is only a couple days old, so it probably already has the fix we needed from the prerelease build. if that's the case, we can just unpin this again

Reply:
just unpinned